tests/unit/local/lambdafn/test_runtime.py

```diff
@@ -109,7 +106,7 @@ def test_keyboard_interrupt_must_raise(self, LambdaContainerMock):
         self.manager_mock.create.side_effect = KeyboardInterrupt("some exception")
 
         with self.assertRaises(KeyboardInterrupt):
-            self.runtime.create(self.func_config, debug_context=debug_options, event=event)
```

Comment:
You should also remove the `event` var created on line 91

Reply:
resolved